sonic_platform_base/sonic_ssd/ssd_generic.py

```diff
@@ -57,8 +57,9 @@ def __init__(self, diskdev):
                 self.fetch_vendor_ssd_info(diskdev, vendor)
                 self.parse_vendor_ssd_info(vendor)
             else:
-                # No handler registered for this disk model
-                pass
+                # unknown model name , use Virtium to get information
```

Comment:
I think this logic to assume it will be Virtium is a hack. Is there a better way to identify the SSD vendor? We should enhance the existing logic to identify the vendor.

Reply:
I checked the information from SMARTCTL, INNODISK and VIRTIUM in the same device's SSD.
There is no information about vendor. You can see the result from smartctrl that I posted. We can only know "Device Model: TS32XBTMM1600 Serial Number: F318410050". But it can't be a logical statement because it's just a random model name or SN. and there are many this kind of ssd in my work enviorment. The only way I found to find out the correct ssd health and temperature is to use VIRTIUM cmd to check it.